rspirv/sr/structs.rs

```diff
@@ -0,0 +1,7 @@
+use spirv;
```

Comment:
`structs` is not a very descriptive name for this module. It's too general and it implies data representation (which is another representation). Can we have a better one? What about `op`?